components/table/ReportTable/utils.ts

```diff
@@ -152,10 +154,50 @@ export const handlePdfExportClick = (
 ) => {
     const JsPDF = jsPDF
     const doc = new JsPDF()
+    const startX = 14
 
     const groupedEntries = toArray(
         groupBy(timesheetEntries, (entry) => entry.timesheet.id)
     )
+
+    if (groupedEntries.length > 1) {
+        let totalHours = 0
+        const allData: EmployeeHours[] = []
```

Comment:
`allData` is not a very descriptive variable name, try to think of something else.